pkg/util/log/log.go

```diff
@@ -534,6 +534,14 @@ func Tracef(format string, params ...interface{}) {
 	logFormat(seelog.TraceLvl, func() { Tracef(format, params...) }, Logger.tracef, format, params...)
 }
 
+// TracefStackDepth logs with format at the trace level and the current stack depth plus the given depth
+func TracefStackDepth(format string, depth int, params ...interface{}) {
+	currentLevel, _ := GetLogLevel()
+	if currentLevel <= seelog.TraceLvl {
+		TraceStackDepth(depth+2, fmt.Sprintf(format, params...))
```

Comment:
When `TraceStackDepth` gets 1 as depth, it will print the caller's stack, but with this implementation `TracefStackDepth` would expect 0 to do the same, which seems weird.

What do you think of implementing `TracefStackDepth` in a way similar to `TraceStackDepth` (but with the log level check) to avoid this issue ?

Eg.
```go
func InfofStackDepth(format string, depth int, params ...interface{}) {
	currentLevel, _ := GetLogLevel()
	if currentLevel <= seelog.InfoLvl {
		msg := fmt.Sprintf(format, params...)
		Log(seelog.InfoLvl, func() { InfoStackDepth(depth, msg) }, func(s string) {
			Logger.infoStackDepth(s, depth)
		}, msg)
	}
}
```